pkg/plugins/resources/k8s/store.go

```diff
@@ -162,9 +162,30 @@ func (s *KubernetesStore) List(ctx context.Context, rs core_model.ResourceList,
 	if err := s.Converter.ToCoreList(obj, rs, predicate); err != nil {
 		return errors.Wrap(err, "failed to convert k8s model into core counterpart")
 	}
+
+	total, err := s.countK8sResources(ctx, rs)
+	if err != nil {
+		return err
+	}
+	rs.GetPagination().SetTotal(uint32(total))
+
 	return nil
 }
 
+func (s *KubernetesStore) countK8sResources(ctx context.Context, rs core_model.ResourceList) (int, error) {
```

Comment:
Can you check if total elements counting works with different meshes?
Create 2 meshes: "default" and "demo"
Create 2 traffic permissions in "default" mesh and in 2 in demo
List traffic permissions by "demo" and see if the total is 2 or 4.

Reply:
Right, so I need additional filtering when counting the resources here.